app/javascript/dashboard/api/account.js

```diff
@@ -1,9 +1,14 @@
+/* global axios */
 import ApiClient from './ApiClient';
 
 class AccountAPI extends ApiClient {
   constructor() {
     super('', { accountScoped: true });
   }
+
+  createAccount(data) {
+    return axios.post(this.apiVersion + '/accounts', data);
```

Comment:
Please use ES6 template literals.

Reply:
You can initialize the constructor with `accounts` like 
```js
  constructor() {
    super('accounts', { accountScoped: false });
  }
```

In this case, you wouldn't need the method `createAccount`

Reply:
changed to ES6 template literals.
changing the constructer would also affect get accounts.